include/pipettes/core/i2c_writer.hpp

```diff
@@ -33,9 +33,9 @@ class I2CWriter {
 
     template <typename Data>
     requires std::is_integral_v<Data>
-    void write(Data data, uint16_t device_address) {
+    void write(Data data, uint16_t device_address, uint8_t reg = 0x0) {
```

Comment:
Why a default of `0x0`? Why any default?

Reply:
For now I have a default until I've modified the other sensors to match the driver refactor since the `write` command previously did not take in a register. I will add a TODO comment.